src/os/bluestore/NVMEDevice.cc

```diff
@@ -605,9 +614,10 @@ int NVMEManager::try_get(const spdk_nvme_transport_id& trid, SharedDriverData **
             probe_queue_cond.wait(l);
           }
         }
+        for (auto p : probe_queue)
+          p->done = true;
```

Comment:
please add
```c++
probe_queue_cond.notify_all();
```
to notify all waiters.

Reply:
@tchaikov Kefu, thanks for your comments.